client/src/composables/useUser.js

```diff
@@ -1,8 +1,10 @@
 import { reactive, readonly, toRefs, computed } from "@vue/reactivity";
-import { jwtDecrypt, tokenAlive } from "@/helper/jwtHelper";
 import axios from "axios";
-
 import { getbaseURL } from "@/helper/baseUrlHelper";
+import { useJwt } from "@vueuse/integrations/useJwt";
+
+// Enables helpfull logs to understand auth
+const DEBUG = false;
```

Comment:
Willst du das bevor wir in Production gehen überall entfernen? Oder wie ist das angedacht? Ansonsten könnte man das ja in eine env-var geben und daraus auslesen.

Reply:
ja, genau, das hätte ich dann entfernt. wobei es jetzt auch nicht wirklich stören würde